ts/components/messages/MessageListComponent.tsx

```diff
@@ -17,6 +17,10 @@ type OwnProps = {
   onPressItem: (id: string) => void;
   onLongPressItem: (id: string) => void;
   selectedMessageIds: Option<Set<string>>;
+  ListEmptyComponent?:
+    | React.ComponentType<any>
```

Comment:
you can probably avoid repeating these types and refer to the `FlatList` prop type like `ComponentProps<typeof FlatList>["ListEmptyComponent"]`